Add documentation for Bryant Evolution integration. (#33292)
home-assistant/home-assistant.github.io: source/_integrations/bryant_evolution.markdown

---
title: Bryant Evolution
description: Instructions for how to integrate a Bryant Evolution HVAC system using a System Access Module.
ha_category:
  - Climate
featured: false

> ha_release: 2024.???

Reviewer: ```suggestion
ha_release: 2024.8
```

ha_iot_class: Local Polling
ha_config_flow: true
ha_domain: bryant_evolution
ha_platforms:
  - climate
ha_zeroconf: false
ha_homekit: true
ha_integration_type: integration
---

The **Bryant Evolution** {% term integration %} lets you control Bryant Evolution HVAC systems.

> 

Reviewer: ```suggestion

## Prerequisites

```

> It requires that a System Access Module be connected to the HVAC system (e.g., a Bryant SYSTXBBRCT01).

Reviewer: ```suggestion
 - It requires that a System Access Module be connected to the HVAC system (for example, a Bryant SYSTXBBRCT01).
```

To install:

1. Connect the System Access Module to a computer using an RS-232-to-USB adapter.

> 1. Download [evolutionhttp](https://github.com/danielsmyers/evolutionhttp). Build and run the server binary, passing --device=/dev/ttyUSB0 (or whatever device your adapter is connected to).

Reviewer: ```suggestion
2. Download [evolutionhttp](https://github.com/danielsmyers/evolutionhttp). 
3. Build and run the server binary, passing --device=/dev/ttyUSB0 (or whatever device your adapter is connected to).
```

> 1. Install the bryant_evolution integration in Home Assistant, passing the IP or hostname of the computer to which the System Access Module is connected.

Reviewer: ```suggestion
1. Install the bryant_evolution integration in Home Assistant, passing the IP or hostname of the computer to which the System Access Module is connected.

{% include integrations/config_flow.md %}

```

> 1. Install the bryant_evolution integration in Home Assistant, passing the IP or hostname of the computer to which the System Access Module is connected.

Reviewer: ```suggestion
4. Install the bryant_evolution integration in Home Assistant, passing the IP or hostname of the computer to which the System Access Module is connected.
```

---
title: Bryant Evolution
description: Instructions for how to integrate a Bryant Evolution HVAC system using a System Access Module.
ha_category:
  - Climate
featured: false
ha_release: 2024.8
ha_iot_class: Local Polling
ha_config_flow: true
ha_domain: bryant_evolution
ha_platforms:
  - climate
ha_zeroconf: false
ha_homekit: true
ha_integration_type: integration
---

The **Bryant Evolution** {% term integration %} lets you control Bryant Evolution HVAC systems.

## Prerequisites

- It requires that a System Access Module be connected to the HVAC system (for example, a Bryant SYSTXBBRCT01).

To install:

1. Connect the System Access Module to the device running Home Assistant, e.g., using an RS-232-to-USB adapter.

> 2. Install the bryant_evolution integration in Home Assistant, passing the name of the serial port to which the device is connected (e.g., `/dev/ttyUSB0`)

Reviewer: in a previous review, I added the config_flow below here. Is there a particular reason this was removed? Does it not apply?
{% include integrations/config_flow.md %}

---
title: Bryant Evolution
description: Instructions for how to integrate a Bryant Evolution HVAC system using a System Access Module.
ha_category:
  - Climate
featured: false
ha_release: 2024.8
ha_iot_class: Local Polling
ha_config_flow: true
ha_domain: bryant_evolution
ha_platforms:
  - climate
ha_zeroconf: false
ha_homekit: true
ha_integration_type: integration
---

The **Bryant Evolution** {% term integration %} lets you control Bryant Evolution HVAC systems.

## Prerequisites

- It requires that a System Access Module be connected to the HVAC system (for example, a Bryant SYSTXBBRCT01).

To install:

> 1. Connect the System Access Module to the device running Home Assistant, e.g., using an RS-232-to-USB adapter.

Reviewer: ```suggestion
1. Connect the System Access Module to the device running Home Assistant, for example, by using an RS-232-to-USB adapter.
```

> 2. Install the bryant_evolution integration in Home Assistant, passing the name of the serial port to which the device is connected (e.g., `/dev/ttyUSB0`)

Reviewer: ```suggestion
2. Install the Bryant Evolution integration in Home Assistant, passing the name of the serial port to which the device is connected (for example, `/dev/ttyUSB0`)
```

> {% include integrations/config_flow.md %}

Reviewer: Below here, please add a section called Climate, providing a bit of detail what this device can do. For example, like the Gree Climate integration docs. 
If there are any supported models, please add a section listing those.
